src/main/diagnostics/diagnostics.ts

```diff
@@ -0,0 +1,48 @@
+import yaml from 'yaml';
+
+import DIAGNOSTICS_TABLE from '@/assets/diagnostics.yaml';
+
+type DiagnosticsFix = {
```

Comment:
Does `DiagnosticsFix` need to be its own type?

Reply:
Yes, because more fields are likely to appear. For example, and ID that can be used to execute a function to try to fix the problem.